helm-charts/common/tgi/rocm-values.yaml

```diff
@@ -2,25 +2,25 @@
 
 accelDevice: "rocm"
 image:
-  repository: ghcr.io/huggingface/text-generation-inference
-  tag: "2.4.1-rocm"
+  repository: opea/tgi-rocm
+  tag: latest
 MAX_INPUT_LENGTH: "1024"
 MAX_TOTAL_TOKENS: "2048"
 USE_FLASH_ATTENTION: "false"
 FLASH_ATTENTION_RECOMPUTE: "false"
 HIP_VISIBLE_DEVICES: "0"
 MAX_BATCH_SIZE: "4"
 extraCmdArgs: ["--num-shard","1"]
+tokenizerMountPath: /usr/src/out
 resources:
   limits:
     amd.com/gpu: "1"
   requests:
     cpu: 1
     memory: 16Gi
 securityContext:
-  readOnlyRootFilesystem: false
-  runAsNonRoot: false
```

Comment:
If the `ghcr.io/huggingface/text-generation-inference:2.4.1-rocm` do not support runAsNonRoot, why not just keep it as `false`, run with root, until tgi solve this issue? Any other concern about this setting? @chyundunovDatamonsters